code/modules/mob/inventory.dm

```diff
@@ -337,6 +337,7 @@
 		I.pixel_x = I.base_pixel_x + rand(-6, 6)
 		I.pixel_y = I.base_pixel_y + rand(-6, 6)
 	I.do_drop_animation(src)
+	return I
```

Comment:
You need to update the doc comment of this now that the return value changed

Reply:
Also because of the `. = doUnEquip`, this proc returns `null | FALSE | /obj/item`

Reply:
think i fixed it